src/PatchOrchestrationApplication/NodeAgentNTService/src/Utility/OperationResultFormatter.cs

```diff
@@ -146,6 +148,7 @@ class WUUpdateWrapper
         public IUpdate2 Update { get; private set; }
         public bool IsDownloaded { get; set; }
         public bool IsInstalled { get; set; }
+        public int HResult { get; set; } = 0;// Should be set in a way that it is null also.
 
         public WUUpdateWrapper(IUpdate2 update, bool isDownloaded, bool isInstalled)
```

Comment:
>WUUpdateWrapper [](start = 15, length = 15)

Is this used anywhere? Append HResult?

Reply:
yes in update details..

Reply:
It is used in GetWUCollection which is used to collect all the updates to install. HResult is updated after installation. So, we dont need to append HResult here.